ampadmin.yml

```diff
@@ -0,0 +1,49 @@
+version: "3"
+
+services:
+
+    server:
+        image: ${imageAmp}
+        command: ["adm-server"]
+        deploy:
+            mode: replicated
+            replicas: 1
+            labels:
+                io.amp.role: "infrastructure"
+                io.amp.mapping: "ampadm:31315:tcp:8081"
```

Comment:
we'd better use a port in the high range (like amplifier), this way we'll have more change to go through corporate firewalls.